Rubberduck.Parsing/Grammar/VBA.g4

```diff
@@ -74,7 +74,9 @@
 *   - implemented proper specs for DATELITERAL.
 *   - added comments to parse tree (removes known limitation #2).
 *   - macroConstStmt now allowed in blockStmt.
-*   - allow type hints for parameters.
```

Comment:
Yes, my bad. Why are we keeping a manual changelog btw? Is this for merging with the ANTLR4 grammar repository?